aws_batch/pipeline.py

```diff
@@ -16,7 +16,7 @@
 
 INPUT_BUCKET = 's3://czbiohub-infectious-disease/UGANDA'
 OUTPUT_BUCKET = 's3://yunfang-workdir/id-uganda'
-KEY_S3_PATH = 's3://cdebourcy-test/cdebourcy_7-19-17.pem'
+KEY_S3_PATH = 's3://cdebourcy-test/idseq-alpha.pem'
```

Comment:
since you are changing it, why not put is into a s3 directory which is not cdebourcy-test?